setup.cfg

```diff
@@ -6,7 +6,8 @@ desktop_files=[('share/applications', 'po/uh/', ('content/packages/unknown-horiz
 traverse-namespace=1
 attr=!long,!gui
 [pep8]
-select = W291, W391
+select = W291, W391, E70
 # https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
-ignore = E101,W191
+ignore = E101, W191, W503
```

Comment:
This should be its own commit, in the commit message you can link to the discussion. We don't need an inline comment for this.